emails/payin_bankwire_succeeded.spt

```diff
@@ -1,9 +1,8 @@
-{{ _("The bank wire to your Liberapay wallet has succeeded") }}
+{{ _("The bank wire to your Liberapay wallet cleared") }}
```

Comment:
Payments clear. It makes it more evident what the last step of making a bank wire comes down to. You make it, and it clears, because someone else clears it. (the bank)